src/params/index.ts

```diff
@@ -0,0 +1,23 @@
+export interface TypesParameters {
+  rootTypeName: string
+  filePath: string
+}
+
+export interface Parameters {
+  runtimeEnvName: string
+  types: TypesParameters
+  substitutionPattern: string
+  configPath: string
+  schemaPath: string
+}
```

Comment:
These types could live in the top-level `types.ts` as `Parameters` is imported in many files. Opinions?

Reply:
IMO these are fine where they are @mohoff